### PR TITLE
Instead of compiling binary use latest release for the manual publish workflow

### PR DESCRIPTION
**Pull request by soider**
Use latest public release for publishing docs, instead of compiling by `go run main.go`

Signed-off-by: Mikhail Sakhnov <msakhnov@mirantis.com>

### CODE REVIEW

File: docs/Dockerfile [hunk 1]
```diff
@@ -15,6 +15,8 @@ RUN \
   apk add --no-cache \
     git \
     git-fast-import \
+    curl \
+    jq \
```
Comment (soider) [inline docs/Dockerfile:19]: I don't know when this Dockerfile is used, but if it is used as runner for the publish workflow it is easier to pur k0s binary download process here

File: docs/Makefile [hunk 2]
```diff
@@ -21,13 +21,19 @@ endif
 docs: cli
 	cd .. && mkdocs build --strict
 
+LATEST_RELEASE_META_URL=https://api.github.com/repos/k0sproject/k0s/releases/latest
```
Comment (jnummelin) [inline docs/Makefile:24]: Instead of "latest" shouldn't we use the version given as input for the workflow?

Comment (soider) [reply docs/Makefile:24]: yeah, makes sense

Comment (kke) [reply docs/Makefile]: yes

**Review by jnummelin — APPROVED** [review]

**Pull request merged**